### PR TITLE
code quality tools to cover test code

### PR DESCRIPTION
**Pull request by bailed22**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,14 +8,17 @@ install: ## Install development dependencies
 	poetry install
 
 fmt: install ## (Format) - runs black and isort against the codebase (auto triggered on pre-commit)
-	poetry run black ./dpypelines/*
-	poetry run isort ./dpypelines/*
+	poetry run black ./dpytools/*
```
Comment (SarahJohnsonONS) [inline Makefile]: This should be `poetry run black ./dpypelines/*`. You can also incorporate the tests folder by doing `poetry run black ./dpypelines/* ./tests/*`.

File: Makefile [hunk 2]
```diff
@@ -8,14 +8,17 @@ install: ## Install development dependencies
...
+	poetry run black ./tests/*
+	poetry run isort ./dpytools/*
```
Comment (SarahJohnsonONS) [inline Makefile]: See other comment - use `poetry run isort ./dpypelines/* ./tests/*` instead.

File: Makefile [hunk 3]
```diff
@@ -8,14 +8,17 @@ install: ## Install development dependencies
...
+	poetry run isort ./tests/*
 
 lint: install ## Run the ruff python linter
-	poetry run ruff check ./dpypelines/*
+	poetry run ruff check ./dpytools/*
```
Comment (SarahJohnsonONS) [inline Makefile]: See other comment - use `poetry run ruff check ./dpypelines/* ./tests/*` instead.

File: Makefile [hunk 4]
```diff
@@ -8,14 +8,17 @@ install: ## Install development dependencies
...
+	poetry run ruff check ./test/*
 
 test: install ## Run pytest and check test coverage
-	poetry run pytest --cov-report term-missing --cov=dpypelines
+	poetry run pytest --cov-report term-missing --cov=dpytools
```
Comment (SarahJohnsonONS) [inline Makefile]: Should be `poetry run pytest --cov-report term-missing --cov=dpypelines`

**Review by SarahJohnsonONS — COMMENTED** [review]
Comments added

File: Makefile (hunk 1 above)
Comment (bailed22) [reply Makefile]: Noted

File: Makefile (hunk 3 above)
Comment (bailed22) [reply Makefile]: noted

**Review by NickPapONS — APPROVED** [review]
Looks all good to me, happy the requested changes have been made.

**Pull request merged**